scene/2d/physics/collision_shape_2d.h

```diff
@@ -71,6 +71,8 @@ class CollisionShape2D : public Node2D {
 	void set_disabled(bool p_disabled);
 	bool is_disabled() const;
 
+  bool contains_point(const Vector2 &point);
```

Comment:
```suggestion
	bool contains_point(const Vector2 &p_point) const;
```
Should really be const, and arguments should start with `p_`